Applications/Spire/Source/Ui/InfoTip.cpp

```diff
@@ -0,0 +1,279 @@
+#include "Spire/Ui/InfoTip.hpp"
+#include <QGraphicsDropShadowEffect>
+#include <QGraphicsPathItem>
+#include <QGraphicsScene>
+#include <QGuiApplication>
+#include <QHBoxLayout>
+#include <QMouseEvent>
+#include <QPainter>
+#include <QPainterPath>
+#include <QScreen>
+#include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Ui/Box.hpp"
```

Comment:
The Box class is not used in this class.

Reply:
The basic properties (e.g., BackgroundColor, BorderTopColor) are defined in Box.hpp.